src/client/pages/Offer/index.tsx

```diff
@@ -188,20 +182,23 @@ export const OfferPage = ({
     return <CheckoutSuccessRedirect />
   }
 
-  const onInsuranceSelectorChange = (
+  const handleInsuranceSelectorChange = (
     newSelectedBundleVariant: QuoteBundleVariant,
+    automated: boolean = false,
```

Comment:
Did we have it before or it's something for the future?

Reply:
this parameter? No I added it now to be able to exclude sending analytics events when we change selected variant automatically